src/babelBundle.ts

```diff
@@ -15,10 +15,13 @@
  */
 
 export { types as t } from '@babel/core';
-import { parse } from '@babel/parser';
+import { parse } from '@babel/core';
```

Comment:
Note: The plugin format between `@babel/parser` and `@babel/core` are different, we use the `@babel/core` one in order to provide similar parsing capabilities compared to upstream. We might should re-visit the plugins we add here, so they are aligned [with upstream](https://github.com/microsoft/playwright/blob/f6219e6e793054a89e89868f2277f194586086ea/packages/playwright/bundles/babel/src/babelBundleImpl.ts#L35).